bip-0039/german.txt

```diff
@@ -0,0 +1,2048 @@
+DRAHT
+DRALL
+DRANG
+DRECK
+DREHBUCH
+DREIECK
+DRESSUR
+DROHUNG
+DROMEDAR
+DRUCKER
+DUELL
+DUETT
+DUFT
+DUNG
+DUPLIKAT
+DURST
+DUSCHE
+DUTZEND
+DYNAMIK
+EBBE
+EBENE
+EBER
+ECHOLOT
+ECHTZEIT
+ECKBALL
+ECKE
+EDAMER
+EDELGAS
+EDITION
+EFEU
+EFFEKT
+EGERLING
+EGO
+EHEBETT
+EHEPAAR
+EHRE
+EHRGEIZ
+EIDECHSE
+EIGELB
+EIGNUNG
+EILTEMPO
+EIMER
+EINBRUCH
+EINDRUCK
+EINFUHR
+EINGRIFF
+EINHORN
+EINIGUNG
+EINKAUF
+EINLAGE
+EINNAHME
+EINRAD
+EINSATZ
+EINTOPF
+EINWAND
+EINZAHL
+EISDIELE
+EISEN
+EISFACH
+EISKANAL
+EISTANZ
+EISWEIN
+EISZEIT
+EIZELLE
+EKLAT
+EKSTASE
+ELAN
+ELCH
+ELEFANT
+ELEGANZ
+ELEKTRIK
+ELFMETER
+ELLBOGEN
+ELOQUENZ
+ELSTER
+ELTERN
+EMBLEM
+EMBRYO
+EMISSION
+EMPATHIE
+EMPFANG
+EMPORE
+EMULSION
+ENDLAGER
+ENDOSKOP
+ENDPHASE
+ENDSPURT
+ENERGIE
+ENGPASS
+ENKLAVE
+ENTGELT
+ENTROPIE
+ENTWURF
+ENZIAN
+ENZYM
+EPOCHE
+ERBAUUNG
+ERBFOLGE
+ERBGUT
+ERBPACHT
+ERBSE
+ERDAPFEL
+ERDBODEN
+ERDGAS
+ERDKUGEL
+ERDNUSS
+ERDREICH
+ERDTEIL
+ERDUNG
+ERFINDER
+ERFOLG
+ERGEBNIS
+ERHEBUNG
+ERLASS
+ERLE
+ERNST
+ERNTE
+EROBERER
+ERPEL
+ERREGER
+ERSATZ
+ERTRAG
+ERUPTION
+ERZ
+ESCHE
+ESELSOHR
+ESKAPADE
+ESKORTE
+ESSENZ
+ESSIG
+ESSTISCH
+ESTRICH
+ETAGE
+ETAPPE
+ETAT
+ETHIK
+ETIKETT
+ETUI
+EUTER
+EVIDENZ
+EWIGKEIT
+EXEMPEL
+EXIL
+EXISTENZ
+EXKURS
+EXOT
+EXPANDER
+EXPERTE
+EXPONAT
+EXZESS
+FABRIK
+FACHWERK
+FACKEL
+FADEN
+FAGOTT
+FAHNE
+FAHRRAD
+FAKIR
+FAKTOR
+FALKE
+FALLOBST
+FAMILIE
+FANFARE
+FANPOST
+FANTASIE
+FARBTON
+FASAN
+FASCHING
+FASSADE
+FAULHEIT
+FAUST
+FAZIT
+FEBRUAR
+FEDERUNG
+FEHLER
+FEIERTAG
+FEIGHEIT
+FEILE
+FEIND
+FELDWEG
+FELGE
+FELL
+FELSWAND
+FENCHEL
+FENSTER
+FERIEN
+FERKEL
+FESTUNG
+FETT
+FETUS
+FEUER
+FIASKO
+FIBEL
+FIGUR
+FILET
+FILIALE
+FILZ
+FINALE
+FINDLING
+FINK
+FINNE
+FISKUS
+FLAGGE
+FLANSCH
+FLASCHE
+FLAUTE
+FLECK
+FLEISCH
+FLIESE
+FLINTE
+FLOH
+FLOSKEL
+FLOTTE
+FLUCHT
+FLUGZEUG
+FLUNDER
+FLUR
+FLUSS
+FOHLEN
+FOKUS
+FORELLE
+FORMAT
+FORST
+FORTGANG
+FOTOGRAF
+FOUL
+FRACHTER
+FRAGMENT
+FRAKTUR
+FRAU
+FREGATTE
+FREITAG
+FREQUENZ
+FRESKE
+FREUND
+FRIEDE
+FRISUR
+FROHSINN
+FROSCH
+FUCHS
+FUGE
+FUHRE
+FUNDUS
+FUNGIZID
+FUNKTION
+FURCHT
+FUSSEL
+FUTTER
+GABEL
+GALLE
+GALOPP
+GAMASCHE
+GANS
+GARANTIE
+GARDE
+GARN
+GARTEN
+GARZEIT
+GASHERD
+GASSE
+GASTHAUS
+GASWERK
+GATTUNG
+GAUL
+GAUMEN
+GEBAREN
+GEBIRGE
+GEBOT
+GEBRAUCH
+GEBURT
+GECKO
+GEDANKE
+GEDECK
+GEDULD
+GEFAHR
+GEFILDE
+GEFLECHT
+GEFOLGE
+GEGEND
+GEGNER
+GEHEGE
+GEHIRN
+GEHORSAM
+GEHRUNG
+GEHSTEIG
+GEHWEG
+GEIER
+GEIGE
+GEIST
+GEIZ
+GELATINE
+GELENK
+GELTUNG
+GEMAHL
+GEMEINDE
+GEMISCH
+GENETIK
+GENICK
+GENOM
+GENUSS
+GEOLOGE
+GEPARD
+GERANIE
+GERBEREI
+GEREDE
+GERSTE
+GERUCH
+GESANG
+GESCHENK
+GESETZ
+GESICHT
+GESPANN
+GESTELL
+GETREIDE
+GEWEIH
+GEWINN
+GEZEITEN
+GIEBEL
+GIFTPILZ
+GIGANT
+GILDE
+GIPFEL
+GIPS
+GITARRE
+GITTER
+GLANZ
+GLASUR
+GLATTEIS
+GLAUBE
+GLEIS
+GLOBUS
+GLOCKE
+GLUT
+GNADE
+GNU
+GOCKEL
+GOLDADER
+GOLFCLUB
+GONDEL
+GOTIK
+GOTT
+GRADIENT
+GRAFIK
+GRAMM
+GRANIT
+GRAS
+GRAVUR
+GREIFER
+GREMIUM
+GRENZE
+GRILL
+GRIMASSE
+GRUBE
+GRUFT
+GRUND
+GRUPPE
+GULASCH
+GULLY
+GUMMI
+GUMPE
```

Comment:
Common enough? Didn't know that one. Doesn't mean anything, just leaving this comment so others could react. Written as it's spelled, can't really be written wrongly, so it's fine even if it would be less known.

Reply:
Never heard of that. Sounds like something local, much like "Pömpel" and "Puschen"

Reply:
Word replaced